### DISCUSSION TITLE
GitHub Management - REPOSTING FOR VISIBILITY

### DISCUSSION THREAD

**Discussion by passamo9** [passamo9/LaTeX_EnglandWales_Templates#15] [Q&A]
1. How do I assign contributors to the PR?

2. I still haven't figured out how to see it via Files Changed > ... > View File, so I need to work on that.

3. Maybe we can assign each of you to one of the three projects? Let me know how each of you would like to do this.

4. Would maybe using the project tab be helpful so that I can keep track/see if I need something to review?

**Comment by bond7o6** [comment]
> 2. pdf viewing

![image](https://user-images.githubusercontent.com/69058445/107881094-96641780-6eda-11eb-8686-3f1e0a706395.png)

![image](https://user-images.githubusercontent.com/69058445/107881116-af6cc880-6eda-11eb-9789-5a2b41aeeacc.png)

![image](https://user-images.githubusercontent.com/69058445/107881127-cb706a00-6eda-11eb-86ac-62b55b3c93b7.png)

![image](https://user-images.githubusercontent.com/69058445/107881204-43d72b00-6edb-11eb-9a6b-d7a240c7b483.png)

![image](https://user-images.githubusercontent.com/69058445/107881144-e7740b80-6eda-11eb-8511-4edc36a24744.png)

![image](https://user-images.githubusercontent.com/69058445/107881154-f2c73700-6eda-11eb-965f-30d3449d6366.png)

![image](https://user-images.githubusercontent.com/69058445/107881165-05417080-6edb-11eb-89b1-48ba79709059.png)

Does this work?

**Comment by Giebisch** [comment]
>     1. How do I assign contributors to the PR?

You don't have to assign anyone for a pull request. It works fine like it is, we may consider moving some development things into seperate branches and not discussions in pull requests, but in the end it won't matter.
 
>     3. Maybe we can assign each of you to one of the three projects? Let me know how each of you would like to do this.

Since two out of three are almost finished, I don't think this will be necessary
 
>     4. Would maybe using the project tab be helpful so that I can keep track/see if I need something to review?

I'm not a huge fan, but use whatever you like

**Reply by passamo9** [reply]
>Since two out of three are almost finished, I don't think this will be necessary

Good point.

Also, we have a Contract Template suggestion coming in from reddit, I don't know whether that you have seen that yet.

**Reply by bond7o6** [reply]
> contract

I added a issue to track it, and linked the reddit post. Though a clearer idea of what is wanted should be discussed over there.

**Reply by passamo9** [reply]
Perfect, thank you!

I will upload soon the template
